app/views/admin/users/index.html.erb

```diff
@@ -28,12 +29,14 @@
   <tbody>
     <% @users.each do |user| %>
       <tr>
-        <td class="email">
+        <td class="email" nowrap>
           <%= user.suspended? ? "<del>".html_safe : "" %>
-            <%= link_to "#{user.name} <#{user.email}>", edit_admin_user_path(user) %>
+            <%= link_to "#{user.name}", edit_admin_user_path(user) %>
           <%= user.suspended? ? "</del>".html_safe : "" %>
+          <br><%= user.email %>
```

Comment:
I like these tweaks. User email could probably be set back a bit, and the name emboldened:

![screen shot 2015-01-06 at 18 45 19](https://cloud.githubusercontent.com/assets/319055/5634001/4dd720a6-95d4-11e4-827a-a11992e73413.png)
![screen shot 2015-01-06 at 18 45 39](https://cloud.githubusercontent.com/assets/319055/5634029/62d55d60-95d4-11e4-9abc-7fa774f3fd60.png)